_docs/first-steps.md

```diff
@@ -0,0 +1,126 @@
+</div>
+
+# Basic definitions
+
+## The Docker Engine
+
+The Docker Engine (or just "Docker" as most people refer to it) is a CLI tool
+that allows you to run your applications in a controlled, secure and portable
+environment. As a starting point, you can think of Docker *containers* as
+something similar to virtual machines but with a radically different
+architectural approach that makes them much more efficient and portable.
+
+Starting with Docker will introduce you to lots of different concepts like
+*images* and *containers*. You need to learn all these concepts (or at least
+have a basic idea about them) before you proceed with this page (or any page
+on Portus). Docker provides
+[great documentation](https://www.docker.com/what-docker) for newcomers, so make
+sure to read it and come back for more information.
+
+## Docker Distribution and Portus
+
+Docker Distribution (also known as **Docker Registry**) is a storage and
+distribution solution for your Docker images. It is the backend behind the
+[Docker Hub](https://hub.docker.com/) and it's [open
+source](https://github.com/docker/distribution). The fact that is open source
+means that you can have your own Docker Registry in your own servers. This is
+really interesting from lots of different reasons, but one main thing to
+consider is that it delegates authorization to an "authorization service".
+
+As explained [here](http://localhost:4000/docs/How-to-setup-secure-registry.html)
+one of the main jobs of Portus is being an authorization service for your docker
+registry. The other main goal of Portus is to provide a useful and powerful UI
+on top of your registry. You can learn about all this [here](/features.html).
+
+# Interacting with a registry and Portus in the CLI
+
+## Log in and out
+
+In order to push or pull any image to your registry you will have to login
+first with docker. Assuming that your registry's hostname is
+`https://my.registry:5000`, then you should perform the following in a terminal:
+
+    $ docker login my.registry:5000
+
+The previous command will ask for your credentials and, if everything goes as
+expected, you will successfully login to your registry. Note two things:
+
+1. If you don't specify a hostname, then Docker assumes the
+   [Docker Hub](https://hub.docker.com/).
+2. You don't have to specify the `https://` part.
+
+After logging in your credentials will be stored in your system so you won't
+have to login again unless you explicitly log out with the `docker logout`
+command. Note that in the `logout` command you will also have to pass the
+hostname (or docker will assume Docker Hub).
+
+## Push & pull
+
+The Docker Engine will allow you to push or pull only if you are logged in. If
+this is the case, then it will be up to the authorization service (Portus in
+our case) to decide whether you can push or pull. This depends on which team
+your user belong, which role plays in said team and so on. Check out [this
```

Comment:
Rephrase: "This depends on the team the user belongs to, which role he or she plays in said team and so on."